src/components/CodeSearchResults/CodeSearchResults.test.jsx

```diff
@@ -77,12 +78,14 @@ describe('<CodeSearchResults />', () => {
         .create((
           <MemoryRouter>
             <Provider store={store}>
-              <CodeSearchResults
```

Comment:
[consideration] This might be a good opportunity to DRY up some of these tests, regarding the creating a reusable wrapper for `CodeSearchResults` without having to re-write `<MemoryRouter>`, `<Provider>`, and now, `<IntlProvider>` for each individual test.

Reply:
Nice catch. but i think there are a lot of changes in this PR right now, we should create a separate PR for this